src/python/pants/backend/project_info/rules/dependencies.py

```diff
@@ -45,25 +45,23 @@ class Dependencies(Goal):
 
 @goal_rule
 async def dependencies(
-  console: Console, build_file_addresses: BuildFileAddresses, options: DependenciesOptions,
+  console: Console, addresses: Addresses, options: DependenciesOptions,
 ) -> Dependencies:
-  addresses: Set[str] = set()
+  address_strings: Set[str] = set()
```

Comment:
Thanks for catching this!!

Reply:
MyPy gets the credit for this one :) It doesn't let us redefine the types of variables.